.github/workflows/cd-syftcli.yml

```diff
@@ -94,3 +100,95 @@ jobs:
         run: |
           tox -e syftcli.publish
           twine upload packages/syftcli/dist/*
+
+  create-syftcli-binaries:
+    needs: [deploy-syft-cli]
+    if: always() && (needs.deploy-syft-cli.result == 'success' && ${{needs.deploy-syft-cli.outputs.current_hash != needs.deploy-syft-cli.outputs.previous_hash }}  || github.event.inputs.skip_tests == 'true')
```

Comment:
NITPICK: we could remove github.events.inputs.skip_tests in this work, as it the skip tests affect only the deployment.